test/unit/reductions/grid_search/test_grid_search_regression.py

```diff
@@ -154,8 +154,8 @@ def test_bgl_lagrange_specifications():
 
     tradeoffs = [0, 0.25, 0.5, 0.75, 1]
 
-    target1.fit(X, y, aux_data=A, number_of_lagrange_multipliers=len(tradeoffs))
-    target2.fit(X, y, aux_data=A, lagrange_multipliers=tradeoffs)
+    target1.fit(X, y, sensitive_features=A)
```

Comment:
We should get rid of all usage of A to avoid confusion

Reply:
Agreed. Shall we rename A -> sensitive?

Reply:
I'd go for something a bit more descriptive. `grouping_data` or something like that.

Reply:
it's a test, so we can call it example_sensitive_feature or test_sensitive_feat or something like that?